concordia/static/scss/action-app.scss

```
@@ -9,6 +9,10 @@ body.view-action-app {
     max-height: 100vh;
     display: flex;
     flex-direction: column;
+    font-family: 'Open Sans', sans-serif;
+    border: none;
+    min-width: map-get($container-max-widths, lg);
+    background-color: #fff;
```

This is intended to match the body background color, right? If so, think this should be `body-bg`?

```suggestion
    background-color: $body-bg;
```